src/frontend/frontend.ts

```diff
@@ -12,6 +12,9 @@ import AppTrees from './components/app-trees/app-trees';
 import {Header} from './components/header/header';
 import * as Rx from 'rxjs';
 import {ParseUtils} from './utils/parse-utils';
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
```

Comment:
I just noticed that we need to have `"@angular/forms": "@1.0.0-rc.1"` which haven't been published yet.